src/MusicStore/Controllers/ManageController.cs

```diff
@@ -119,25 +116,25 @@ public IActionResult AddPhoneNumber()
         // POST: /Manage/DisableTwoFactorAuthentication
         [HttpPost]
         [ValidateAntiForgeryToken]
-        public async Task<IActionResult> DisableTwoFactorAuthentication()
+        public async Task<IActionResult> DisableTwoFactorAuthentication(CancellationToken cancellationToken)
```

Comment:
But this token just indicates if request was aborted. Otherwise it will not be clear on what token is passed around here.